Util/Docker/Carla.Dockerfile-mod

```diff
@@ -0,0 +1,39 @@
+FROM carla-prerequisites:latest as build-env
+
+ARG CARLA_REPO=https://github.com/Bonifatius94/carla
+ARG CARLA_BRANCH=keep-scoomatic-resources-during-update
+
+USER carla
+WORKDIR /home/carla
+
+# TODO: don't clone, use the locally checked out repo via ADD command instead
```

Comment:
Do you think that this is possible? 
Currently, we download carla to get the instructions to download carla in the docker image?
Also: I would have to change the arguments every time I want to try something new?

Reply:
Yes it's possible with "ADD . ." if we put all irrelevant files like docs into .dockerignore, so Docker only rebuilds when relevant files change. Then you could build your locally checked out Carla repo without having to push it to GitHub.

Currently the Docker build would assume that "git clone" always returns the same content, so you would need to disable caching with "--no-cache" and rebuild the whole thing anyways.

As said, the build process is really clunky and Docker cannot really make it look elegant. It is what it is 😂